sysl2/sysl/Parser_test.go

```diff
@@ -70,36 +78,71 @@ func parseComparable(filename, root string) (*sysl.Module, error) {
 	return module, nil
 }
 
-func parseAndCompare(filename, root string) (bool, error) {
-	goModule, err := parseComparable(filename, root)
+func parseAndCompare(filename, root, golden string, goldenModule *sysl.Module) (bool, error) {
+	module, err := parseComparable(filename, root)
 	if err != nil {
 		return false, err
 	}
 
-	pyModule, err := pyParse(filename, root)
+	if proto.Equal(goldenModule, module) {
+		return true, nil
+	}
+
+	generated, err := ioutil.TempFile("", "sysl-test-*.textpb")
 	if err != nil {
 		return false, err
 	}
-	if proto.Equal(pyModule, goModule) {
-		return true, nil
-	}
+	defer generated.Close()
+	defer os.Remove(generated.Name())
 
-	golden := "golden.txt"
-	generated := "generated.txt"
-	if err = TextPB(pyModule, golden); err != nil {
+	if err = TextPB(goldenModule, golden); err != nil {
 		return false, err
 	}
-	if err = TextPB(goModule, generated); err != nil {
+
+	if err = FTextPB(generated, module); err != nil {
```

Comment:
`generated` (tempfile) first used here, define right above (I think)

Reply:
Done.

Reply:
I've also done it in `parseAndCompareWithPython`.